source/manual/data-gov-uk-architecture.html.md

```diff
@@ -0,0 +1,126 @@
+
+## Applications
+
+> These are Cloudfoundry applications, show in blue above.
+
+### Legacy DGU
+
+The old platform which Publish and Find are going to replace. It has an API that is accessed to import data on the new platform.
+
+### Find Data
+
+The new public-facing service that end-users access to find data. It’s a Rails app, hosted on [GOV.UK PaaS][paas]. You can find out more about this app [here][find].
+
+### Publish Data
+
+The new publisher-facing services that publishers access to add or edit datasets. It's a Rails app, hosted on [GOV.UK PaaS][paas]. You can find out more about this app [here][publish].
+
+### Publish Data Worker
+
+This is a rails worker used to fetch data from legacy. It uses Redis to queue import tasks. It will be removed once legacy is no longer used. The source code is in the [Publish Data app][publish].
+
+The way data is normally imported from legacy is:
+
+* Every hour, pingdom GETs https://publish-data-beta.cloudapps.digital/api/sync-beta
+* This runs the `sync:beta` Rails task that queries the Legacy API for new and updated datasets
+* Changes are reflected in the Publish database and pushed to elasticsearch
+
+To run the task manually you can do the following on [staging](#staging-environment) (or replace the app name with the live app):
+
+    cf run-task publish-data-beta-staging-worker "bin/rake sync:beta" --name sync
+    cf logs publish-data-beta-staging-worker
+
+## Services
+
+> These are Cloudfoundry services, shown in red above.
+
+| Name | Description | Apps bound |
+| ---- | ----------- | ---------- |
+| beta-production-elasticsearch | The elasticsearch server | publish-data-beta-worker, publish-data-beta, find-data-beta |
+| publish-beta-production-pg | The pg database for Publish | publish-data-beta-worker, publish-data-beta |
+| publish-production-secrets | User-provided service. Provides env variables to Publish | publish-data-beta-worker, publish-data-beta |
+| find-production-secrets | User-provided service. Provides secrets as env variables to Find | find-data-beta |
+| beta-dgu-route | cdn-route. Routes beta.data.gov.uk to find-data-beta | |
+| logit-ssl-drain | TBC | All apps |
+
+### Postgres
+
+The database that [Publish Data][publish] uses. Publish Data gets the details and credentials through the `VCAP_SERVICES` environment variable.
+
+### Elasticsearch
+
+The search index that [Find Data][find] uses to search datasets. It is populated through the `search:reindex` rake task on [Publish Data][publish] (see below) and when publishers make changes when using Publish Data.
+Similarly the `VCAP_SERVICES` environment variable, made available to applications that connect to it, contains the credentials to connect to it.
```

Comment:
This is a bit confusing. How about: 'The VCAP_SERVICES environment variable contains the credentials to connect to it'.